ext/pdo_mysql/tests/bug_61207.phpt

```diff
@@ -19,8 +20,8 @@ $handle1 = $db->prepare('insert into test(id) values(1);
                           select * from test where id = ?;
                           update test set id = 2 where id = ?;');
 
-$handle1->bindValue('1', '1');
-$handle1->bindValue('2', '1');
+$handle1->bindValue(1, '1');
```

Comment:
That's a notable change in the implementation. :/ But maybe a numeric string not too frequently supplied?

Reply:
This looks okay to me.